src/components/pages/PowerheadPage/test.js

```diff
@@ -0,0 +1,235 @@
+		});
+	})
+});
+
+describe('PowerheadPageHTTP', () => {
+
+  it('should be a function', () => {
+	expect(PowerheadPageHTTP).to.be.a('function');
+  });
+
+  it('calling the HTTP should return a request with url', (done) => {
+	const props$ = Rx.Observable.just({});
+	const HTTPSource = Rx.Observable.empty();
+	const {request$, response$} = PowerheadPageHTTP({props$, HTTP: HTTPSource});
+	
+	request$.subscribe(req => {
+
+		expect( _.startsWith(req.url, `${API_PATH}/powerhead/`) ).to.be.true;
+		done();
+	});
+  });
+
+  it('should map the responses to body.results', (done) => {
+	const props$ = Rx.Observable.just({});
+	const HTTPSource = new Rx.ReplaySubject(1);
+	const {request$, response$} = PowerheadPageHTTP({props$, HTTP: HTTPSource});
+	
+	response$.subscribe(response => {
+		expect(response).to.equal("test");
+		done();
+	});
+
+	request$.map(req => {
+		
+		var response$ = Rx.Observable.just({
+			request: req,
+			body: { results: "test" }
+		});
+		response$.request = req;
+		return response$;
+
+	}).subscribe(HTTPSource.asObserver());
+
+
+  });
+
+});
+
+
+describe('PowerheadPage', () => {
+  it('should be a function', () => {
+    expect(PowerheadPage).to.be.a('function');
+  });
+
+  describe('using filter "all"', () => {
+
+  	const props$ = Rx.Observable.just({
+      location: 'all',
+      lookaheadLength: 1,
+      reportLength: 2
+    });
+
+  	const HTTPsource$ = new Rx.ReplaySubject(1);
+
+    const DOMSource = mockDOMResponse();
+
+    const powerHeadPage = PowerheadPage({
+		DOM: DOMSource, 
+		HTTP: HTTPsource$,
+		props$
+	});
+
+    const request$ = powerHeadPage.HTTP;
+
+	request$.map(request => {
+		const mockedResponse$ = Rx.Observable.just({
+	  		body: mockData
+		});
+		mockedResponse$.request = request;
+		return mockedResponse$;
+	})
+	.subscribe(HTTPsource$.asObserver());
+
+    powerHeadPage.DOM.elementAt(0).subscribe(vtree => {
+ 	
+ 		const page = createPowerheadPageObject(vtree);
+
+ 		it('should have a single report', () => {
+ 			expect(page.reportCount).to.equal(1);
+ 		});
+
+ 		it('should have a sum of internals from all reports, for each month', () => {
+ 			expect(page.internals).to.eql(['30','30','30']);
+ 		});
+
+ 		it('should have a sum of bench from all reports, for each month', () => {
+ 			expect(page.bench).to.eql(['5','5','5']);
+ 		});
+
+ 		it('should have a sum of booked from all reports, for each month', () => {
+ 			expect(page.booked).to.eql(['25','25','25']);
+ 		});
+
+ 		it('should have a sum of externals from all reports, for each month', () => {
+ 			expect(page.externals).to.eql(['3','3','3']);
+ 		});
+
+		it('should have a sum revenue from all reports, for each month', () => {
+ 			expect(page.confirmed_revenue).to.eql(['3 000', '3 000', '3 000']);
+ 		});
+
+ 		it('should have a sum overruns from all reports, for each month', () => {
+ 			expect(page.overruns).to.eql(['15','15','15']);
+ 		});
+
+    });
+
+
+  });
+});
```

Comment:
Overall, great tests. Thanks.
We just need to fix some ESLint issues and that's it.